t/10-tests_overview.t

```diff
@@ -237,17 +237,21 @@ $t->element_exists('#res_DVD_i586_kde');
 $t->element_exists('#res_GNOME-Live_i686_RAID0 .state_cancelled');
 $t->element_exists('#res_NET_x86_64_kde .state_running');
 
-subtest 'incomplete results generally accounted as "Incomplete"' => sub {
+subtest 'not complete results generally accounted as "Incomplete"' => sub {
     my $schema = $t->app->schema;
     $schema->txn_begin;
 
     my $jobs = $schema->resultset('Jobs');
+
+    # INCOMPLETE and TIMEOUT_EXCEEDED are expected to be accounted as incomplete
     $jobs->search({id => 99937})->update({result => OpenQA::Jobs::Constants::INCOMPLETE});
-    $jobs->search({id => 99764})->update({result => OpenQA::Jobs::Constants::USER_CANCELLED});
     $jobs->search({id => 99946})->update({result => OpenQA::Jobs::Constants::TIMEOUT_EXCEEDED});
 
+    # USER_CANCELLED is not expected to be accounted as incomplete
+    $jobs->search({id => 99764})->update({result => OpenQA::Jobs::Constants::USER_CANCELLED});
```

Comment:
ok but would we not have the same problem in the overview as for incompletes? Imagine a build that consists of a single "cancelled" job , it will not be counted? Or are you fixing a problem that it was counted as "None" in before but additionally as "Incomplete"?

Reply:
https://github.com/os-autoinst/openQA/pull/2564 changed the behavior so `USER_CANCELLED` is accounted as `INCOMPLETE_RESULTS` (previously it was just counted as 'None'). We said that `USER_CANCELLED` actually belongs into the new meta result `ABORTED_RESULTS`. This test change reflects that change.